UOP1_Project/Assets/Scripts/Editor/GameSceneEditor.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+[CustomEditor(typeof(GameScene), true)]
```

Comment:
Where is `GameScene` class?

Reply:
`GameScene` was already present in this **Scene-Loading** branch on Scripts/Scriptable Objects/SceneManagement before I started working on this PR